src/store/modules/errors.ts

```diff
@@ -56,7 +56,7 @@ export default {
         [ERROR_MESSAGE]: (state: State): string | null => {
             const error = state.errors.length > 0 ? state.errors[0] : null;
             if (error == null) return null;
-            if (error instanceof Error) return error.stack || "";
+            if (error instanceof Error) return error.toString() || "";
```

Comment:
would be nice to get the console.warn as we discussed Friday
`if (error instanceof Error) {
                console.warn(error.toString());
                return error.toString() || "";
            }`